### ISSUE TITLE
Simplify directory structure

### ISSUE THREAD

**Issue by anandthakker** [developmentseed/project-seed#52]
I think the directory structure here is deeper/more complicated than necessary.  Here's what I'd propose:

```
. (project root)
|-- LICENSE
|-- README
|-- gulpfile.js
|-- package.json
`-- scripts/ (javascript sources -- compiled into build/bundle.js)
|   `-- lib/
|   `-- components/
|   etc.
`-- styles/ (scss sources -- compiled into build/main.css)
`-- assets/ (any static assets -- copied directly into build/)
|   |-- index.html
|   |-- humans.txt
|   |-- robots.txt
|   `-- graphics/
|   `-- data/
`-- tests/
```

**Comment by anandthakker** [comment]
Update - added `tests/`

**Comment by dereklieu** [comment]
@danielfdsilva have any thoughts here? Working in IFPRI-egypt now and I'm finding it a bit cumbersome to have such a deeply nested directory structure. I'd be up for simplifying this if at all possible.

**Comment by danielfdsilva** [comment]
@dereklieu I agree that our directory structure is a bit deep but it was to keep things organized. If we feel like it's too complicated nothing against revisiting.

On my side I've a couple of notes:
- I like that we have all the app related stuff inside a folder. Makes it easy to see what refers to the site and what's build automation
- It would be good that the structure of the `src` matches the `build`. Makes it easy to know where the files end up and how to construct urls. Otherwise you'd have to interpret the build file to know where the files are compiled to.
- This is a direct comment to the structure above. Files like `index.html` are not exactly assets, they should be elsewhere.

@ricardomestre probably has some input as well.

**Comment by ricardomestre** [comment]
I might suffer from mad OCD, but I agree with @danielfdsilva here. I find way more pleasant and easy to browse and manage a project when the source folder **structure** kind of matches the **build**. In that sense, I would actually rename `app` to `source`.
